pyblazing/pyblazing/apiv2/context.py

```diff
@@ -2388,8 +2401,10 @@ def _optimize_skip_data_getSlices(
                 file_and_rowgroup_indices = (
                     file_indices_and_rowgroup_indices.to_pandas()
                 )
-                grouped = file_and_rowgroup_indices.groupby("file_handle_index")
 
+                grouped = file_and_rowgroup_indices.groupby(
```

Comment:
This does not make sense. Can you explain this? This used to be groupby a column name, now its group by a column of values?